lib/generators/rspec/authentication/templates/session_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'rails_helper'
+
+RSpec.describe "Sessions", <%= type_metatag(:request) %> do
+  fixtures :users
+
+  let(:user) { users(:one) }
```

Comment:
Although we do generate fixtures when thats enabled, we don't use them anywhere else, I feel like this should be gated on fixtures being turned on but I'm not sure what the alternative should be?